### ISSUE TITLE
Setting up the points

### ISSUE THREAD

**Issue by sam-chapp** [Cacti/plugin_gpsmap#9]
I have added the api key and link and have put the longitude and latitude of our location we want but its not showing has anyone got any setup guides or tutorials?

**Comment by netniV** [comment]
You must make sure to add the device template that you wish to appear on the map to Templates -> Map

**Comment by sam-chapp** [comment]
> You must make sure to add the device template that you wish to appear on the map to Templates -> Map

I have done that however it still doesnt show what would the Group ID be?

**Comment by netniV** [comment]
There isn't much to this plugin.  

1. make sure you have lng/lat on your device
2. make sure the device template is in the table of desired templates for the map
3. make sure the poller has been running and generated the maps.

That last one has proven tricky as at the moment you don't get stats in the cacti log for this plugin.  However, you should get output in the XML/ directory.  If you want to make sure the poller is running, remove it all and it should recreate it.

**Comment by sam-chapp** [comment]
https://puu.sh/CUVTP/334d61457c.png
https://puu.sh/CUVUE/59ef7e10b2.png 

Im not too sure about this: "make sure the poller has been running and generated the maps." XML ?? Do you have any screenshots?

**Comment by netniV** [comment]
Do you have any data in your XML folder?

**Comment by netniV** [comment]
If not, make sure the website has write permissions to that folder (if you have SELinux enabled, you will likely have to make sure the correct context has been set to).

**Comment by sam-chapp** [comment]
I see the XML folder there is just an index.php file

**Comment by sam-chapp** [comment]
Is it the file inside ""/usr/share/cacti/plugins/gpsmap/"" ?

**Comment by netniV** [comment]
OK, that would suggest that either you haven't enabled the maps plugin fully, or it doesn't have write access to the XML folder when running as the poller user. (normally run from /etc/cron.d/cacti)

**Comment by sam-chapp** [comment]
> OK, that would suggest that either you haven't enabled the maps plugin fully, or it doesn't have write access to the XML folder when running as the poller user. (normally run from /etc/cron.d/cacti)

I have made the folder 777 perm and have set the cacti to 777 in cron.d

**Comment by netniV** [comment]
Yeah, you didn't need to do that from cron.d's cacti file.  It reads it as a configuration file, not to run as a script.

So if your permissions are there and cacti is polling, do you have SELinux enabled?

**Comment by sam-chapp** [comment]
> Yeah, you didn't need to do that from cron.d's cacti file. It reads it as a configuration file, not to run as a script.
> 
> So if your permissions are there and cacti is polling, do you have SELinux enabled?

I believe we just have a basic centos 6 server

**Comment by Eduardo251** [comment]
hello i do have the same issue however cacti is generating more files in the XML folder but it isnt displaying anithing in the map page but a singe ip  

her are some screenshots:
snipboard.io/jSMsP8.jpg
snipboard.io/johza3.jpg

**Comment by TheWitness** [comment]
You might have to help yourself on this one.  I'm not using it as the API and key requirements drive me nuts.  If you figure things out, submit a pull request.
